docs/lava-job-priority.md

```diff
@@ -19,9 +19,8 @@ requirements, and it fully takes advantage of having 101 priorities available.
   - stable-rc sanity jobs (all branches)
   - mainline sanity jobs
 - 79: [Developer builder](developer-builder.md) jobs
-- 78: stable branches which are not LTS (e.g. 5.x)
-- 78: stable-rc 5.6
-- 77: stable-rc 5.5
+- 78: stable-rc 5.7
+- 77: stable branches which are not LTS (e.g. 5.x)
```

Comment:
Shouldn't this be kept as 78?

Reply:
@mrchapp Done.
Your review comments addressed.